### PR TITLE
Task 90 Add copyright headers to all source files

### PR DESCRIPTION
**Pull request by mmews-n4**
Adds copyright headers. See #90.

### CODE REVIEW

File: addCRH.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```
Comment (LorenzoBettini) [inline addCRH.sh]: You may want to add a copyright to this very file as well :)
Please, feel free to add your name as the contributor

Comment (mmews-n4) [inline addCRH.sh]: Oh, my mistake. The script wasn't intended to be added to your repo; I'll remove it from the branch again. However, feel free to use it.

**Comment by LorenzoBettini** [comment]
The build fails due to test expectations that have to be updated. I think the generation tests that fail are the ones where the expectations are searched for by using files in the "expectations" folder according to "output" folder. Thus, files should be copied into "expectations" folder. If you don't feel like doing that, I can take care of that, but I don't know when :)

**Comment by LorenzoBettini** [comment]
I'm taking care of updating tests expectations in this PR https://github.com/LorenzoBettini/xsemantics/pull/92 which is based on your PR.

**Comment by mmews-n4** [comment]
Ok, thanks!

**Pull request merged**